USER_MANUAL.md

```diff
@@ -336,6 +363,345 @@ _TODO_
+
+**Notes on acquisition:** Users are encouraged to undertake an exercise to assess whether it's logical for a particular source to generate multiple emissions simultaneously.
+
+**Notes of caution:** When set to false, it's possible to observe lower emission production rates than what has been parameterized, as existing emissions inhibit the generation of new emissions.
+
+#### &lt;NR_ERS&gt;
+
+**Data Type:** String
+
+**Default input:** The name of a column in the [emissions file](#emissions_file) that is used to parameterize the emissions characteristics of all the general non-repairable emissions. This parameter will be overwritten by the equivalent value, when provided at a more granular scale.
+
+**Notes on acquisition:** `NR_ERS` serves as the counter part to [ERS](#ers) for non-repairable emissions.
+
+**Notes of caution:** The column headers are case sensitive.
+
+#### &lt;NR_EPR&gt;
+
+**Data Type:** Float (Numeric)
+
+**Default input:** None
+
+**Description:** A numeric scalar representing the non-repairable emissions production rate (NR_EPR), which denotes the emission production rate for non-repairable emissions. New non-repairable emissions are generated using a site-level empirical NR_EPR. NR_EPR is the probability that a new non-repairable emission will arise, each day, for each site.
+
+**Notes on acquisition:** `NR_EPR` serves as the counter part to [LPR](#lpr) for non-repairable emissions. Refer to [LPR](#lpr) for more details.
+
+**Notes of caution:** When the parameter [NR_multi](#nr_multi) is set to false, it will affect the non-repairable emission production rate observed in the simulation. In this scenario, new emissions won't be generated if there's already an existing emission for the specified emission source.
+
+#### &lt;duration&gt;
+
+**Data Type:** Integer (Numeric)
+
+**Default input:** 365
+
+**Description:** The duration or number of days of each of the non-repairable emission lasts. Represents emission removal from the non-repairable emission pool.
+
+**Notes on acquisition:** `duration` serves to act as the counter part of [NRd](#nrd) for non-repairable emissions.
+
+**Notes of caution:** N/A
+
+#### &lt;NR_multi&gt;
+
+**Data Type:** Boolean
+
+**Default input:** False
+
+**Description:** Specifies whether a non-repairable emission source can generate multiple emissions simultaneously. For instance, a flare that's unlit can't produce additional unlit emissions simultaneously.
+
+**Notes on acquisition:** `NR_multi` serves to act as the counter part of [multi](#multi) for non-repairable emissions.
+
+**Notes of caution:** When set to false, it's possible to observe lower emission production rates than what has been parameterized, as existing emissions inhibit the generation of new emissions.
+
+### &lt;Repairs&gt;
```

Comment:
Once again with the repair parameters - Need to talk about them being propagating